packages/jest-cli/src/watch.js

```diff
@@ -167,21 +150,54 @@ const watch = (
         startRun({updateSnapshot: true});
         break;
       case KEYS.A:
-        setWatchMode(argv, 'watchAll');
+        setWatchMode(argv, 'watchAll', {
```

Comment:
Shall we rename this function to "setWatchState", or given that it is inside of "watch.js", simply call it "setState"? I think that makes more sense.